### PR TITLE
Prevent evaluating stale output node.

### PR DESCRIPTION
**Pull request by parvizp**
When quantizing to 8-bit the Identity ops are removed ("spliced"). For graph's with a parameter-less output node like SoftMax, there is no issue. However, if the graph's output node received a constant input (i.e. a bias add with weights) the output node referenced [here](https://github.com/parvizp/tensorflow/blob/502ecd2c6c324a1c5c3f7b195c36dbafb45dafb3/tensorflow/contrib/quantization/tools/quantize_graph.py#L318) will be stale (still has Identity input) and result in a error.

### CODE REVIEW

**Comment by parvizp** [comment]
I signed it!
The cooperation is NanoWatt Design.

**Comment by danmane** [comment]
We should get the CLABot happy before we go forward with this CL. @parvizp can you try poking it again?

**Comment by parvizp** [comment]
I signed it!

I had the email address used on the commits and CLA as secondary email on my GitHub profile. I changed to be primary in case that helps.

**Comment by parvizp** [comment]
I also need to write a test.

**Comment by rmlarsen** [comment]
@parvizp Any update?

**Comment by vrv** [comment]
Closing due to lack of CLA + lots of time passing
